src/js/nav/components/NavBar.js

```diff
@@ -74,11 +83,16 @@ export const Bar = ({ administrator, dev, userId, onLogout }) => (
 
             <Dropdown>
                 <NavDropdownButton>
-                    <Icon name="user" />
+                    <InitialIcon handle={handle} size="md"></InitialIcon>
+
                     <span>{userId}</span>
                     <Icon name="caret-down" />
                 </NavDropdownButton>
                 <DropdownMenuList>
+                    <DropdownMenuLink to="/account">
+                        Signed in as <b>{handle}</b>
```

Comment:
```suggestion
                        Signed in as <strong>{handle}</strong>
```

Both should act identically in standard browsers, but can behave differently with assistive technology and non-standard browsers as `<b>` is strictly styling while `<strong>` indicates semantic importance